### ISSUE TITLE
Blank line between "class" and "def" lines

### ISSUE THREAD

**Issue by martemyev** [PyCQA/pycodestyle#712]
Hi All,

Could you please clarify if the following piece of code violates PEP8 and pycodestyle should report an error:

```py
class A:
    def func():
        pass
```

I haven't found an example of this in the testsuite, and pycodestyle doesn't report any errors on this, though PEP8 states:
> Method definitions inside a class are surrounded by a single blank line.

Thanks,
Mikhail

**Comment by sigmavirus24** [comment]
What version of pycodestyle are you using?

**Comment by martemyev** [comment]
I just tried it with the most recent version from the 'master' branch. Same result.

**Comment by martemyev** [comment]
Any thoughts on this issue?

**Comment by adiroiban** [comment]
From pep8

```
Blank lines may be omitted between a bunch of related one-liners (e.g. a set of dummy implementations).
```

I have tested with latest eb91b79 and I can confirm this.

```
(build) $ pycodestyle -
# Some comment here


def some_function():
    pass


class A:
    def func():
        pass
(build) $
```

I would say that pycodestyle should complain, as is hard to tell which is a dummy and which is not... and adding extra spaces will not hurt :)

**Comment by asottile** [comment]
Duplicate of #438
